src/shared/types/input.types.ts

```diff
@@ -0,0 +1,12 @@
+export interface InputPropsTypes {
```

Comment:
These are not going to be used project wise, move it with component

Reply:
done it by initilizing it in the component